hasura/migrations/1607542601049_alter_table_public_Guild_add_column_join_button_url/down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "public"."Guild" DROP COLUMN "join_button_url";
```

Comment:
Could you squash these ADD/DROP columns migrations?

Reply:
So I recently was on a call with Hasura and they said they're starting to recommend people to not squash migrations since it can start to become problematic and you lose the ability to see where things went wrong. For these few it's not a problem, but yea in general I want to start avoiding squashing unless we are sure it's not going to cause problems like these new columns we are adding

Reply:
fair enough

Reply:
I do not see clearly where that can be problematic.
Here we do a pull request, that will affect the production only once deployed, and should be able to be rolled back 
Maybe they meant squashing migration that are already deployed ?

I find it very confusing like this, hard to review, without saying that we will endup with an enourmous list of migration over time